frontend/src/components/RecentOperations/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useCallback, useEffect } from "react";
+import { useDispatch } from "react-redux";
+import { Table } from "@stellar/design-system";
+
+import { useRedux } from "hooks/useRedux";
+import { networkConfig } from "constants/settings";
+import { SectionCard } from "components/SectionCard";
+import { AccountBadge } from "components/AccountBadge";
+import { OperationTypeColumn } from "./OperationTypeColumn";
+import { fetchLastOperationsAction } from "ducks/operations";
+import { FetchLastOperationsActionResponse, Network } from "types";
+
+import "./styles.scss";
+
+export const RecentOperations = () => {
+  const { operations } = useRedux("operations");
+  const dispatch = useDispatch();
+  const horizonURL = "https://horizon.stellar.org";
+
+  const labels = [
+    {
+      id: "source",
+      label: "Source",
+    },
+    {
+      id: "operations",
+      label: "Operations",
+    },
+    {
+      id: "details",
+      label: "Details",
+    },
+    {
+      id: "timeAgo",
+      label: "Time ago",
+    },
+  ];
+
+  const getLastOperations = useCallback(() => {
+    dispatch(fetchLastOperationsAction());
+  }, [dispatch]);
+
+  useEffect(() => {
+    getLastOperations();
+    setInterval(getLastOperations, 10 * 1000);
+  }, [dispatch, getLastOperations]);
+
+  const renderOperations = (operation: FetchLastOperationsActionResponse) => {
+    const operationLink = `${horizonURL}/operations/${operation.id}`;
+
+    return (
+      <React.Fragment key={Math.random()}>
+        <td>
+          <AccountBadge
+            id={operation.source_account}
+            horizonURL={horizonURL}
+            account=""
+          />
+        </td>
+        <td>
+          <a
+            className="RecentOperations__link"
+            href={operationLink}
+            target="_blank"
+            rel="noreferrer"
+          >
+            {operation.type}
+          </a>
+        </td>
+        <td>
+          <OperationTypeColumn horizonURL={horizonURL} operation={operation} />
```

Comment:
If possible, could we truncate (add `...`) details if it doesn't fit on one line? We could use CSS for that. This could be an edge case, but it will be nice to ensure it won't break the UI.